### DISCUSSION TITLE
tina cms error while deleting any .md file

### DISCUSSION THREAD

**Discussion by Manavbhavsar-4703** [tinacms/tinacms#4423] [Help]
![Screenshot 2024-01-12 163447](https://github.com/tinacms/tinacms/assets/156056255/dcf064b0-8853-4145-86c1-77a766983380)
![Screenshot 2024-01-12 163824](https://github.com/tinacms/tinacms/assets/156056255/9fa1b9ff-0c78-4562-b7ad-3bdfcaf4bb72)
I m using astro with tinacms ,while deleting .md file form collections i m getting this error pls help

**Comment by kldavis4** [comment]
This looks like a duplicate of #4018

**Reply by Manavbhavsar-4703** [reply]
can u pls tell me solution for this error ??

**Reply by kldavis4** [reply]
It's a bug - the workaround is to manually delete the file or use something other than windows for local dev. (Note: this is not an issue in production environments)
